services/src/main/java/org/exoplatform/officeonline/WOPIService.java

```diff
@@ -361,6 +381,12 @@ public void start() {
     }
     discoveryPlugin.start();
 
+    if (activeCache.get(SECRET_KEY) == null) {
+      String secretKey = generateSecretKey();
+      LOG.debug("Generated secret key: " + secretKey);
+      activeCache.put(SECRET_KEY, secretKey);
```

Comment:
As we use cache to store secret keys, we should care about eviction and our logic behaviour: need configure this cache to work w/o eviction, thus the keys always will be cached.

Reply:
Fixed in: https://github.com/exo-addons/office-online/pull/2/commits/a0bd2084c19c9658b8aa5a48314c1653208a7876